plugin/src/main/java/com/denizenscript/denizen/objects/ChunkTag.java

```diff
@@ -292,6 +292,9 @@ public static void register() {
         // @returns ChunkTag
         // @description
         // Returns the chunk with the specified coordinates added to it.
+        // @example
+        // # Notes the chunk with 10 added onto the X and Z axes of the chunk, making the coordinates "15,15,world".
+        // - note <chunk[5,5,world].add[10,10].cuboid> as:my_new_chunk_cuboid
```

Comment:
The cuboid noting is a bit odd for demonstrating basic chunk usage. I'd encourage chunk-related features like `chunkload` command, ChunkTag mechs, ...